app/views/assessments/_edit_basic.html.erb

```diff
@@ -20,8 +20,7 @@
 <h4>Modules Used</h4>
 <div class="form-group">
   <% if @assessment.has_autograder? %>
-    <%= link_to "Edit Autograder Settings", [:edit, @course, @assessment, :autograder], class: "btn primary" %>
-    <%= link_to "Delete Autograder", [@course, @assessment, :autograder], method: :delete, class: "btn danger",
+    <%= link_to "Remove Autograder", [@course, @assessment, :autograder], method: :delete, class: "btn danger",
```

Comment:
If you're gonna change this from "Delete" to "Remove", you should do the same on the actual autograder form as well. Same for the Scoreboard.